uint64 token
uint64 program_id
# limit on number of outputs, or 0 for all
uint64 limit
---
Output[] output
